sos/plugins/kubernetes.py

```diff
@@ -182,4 +163,31 @@ def postproc(self):
         # output that is not hit by the previous iteration.
         self.do_cmd_private_sub('kubectl')
 
+
+class RedHatKubernetes(Kubernetes, RedHatPlugin):
+    """Red Hat Kubernetes plugin
+    """
+
+    # OpenShift Container Platform uses the atomic-openshift-master package
+    # to provide kubernetes
+    packages = ('kubernetes', 'kubernetes-master', 'atomic-openshift-master')
+
+    files = (
+        '/etc/origin/master/admin.kubeconfig',
+        '/etc/origin/node/pods/master-config.yaml',
+    )
+
+    kube_cmd = "kubectl --kubeconfig=/etc/origin/master/admin.kubeconfig"
```

Comment:
This config file is only valid for OCP deployments of kubernetes. While this is the only type of deployment that RH currently supports, it is possible to run "plain" kubernetes on RHEL, in which case the `kubectl` commands will fail with this `--kubeconfig` as that file won't be present.